src/dist/mod.rs

```diff
@@ -45,7 +45,7 @@ pub static DEFAULT_DIST_SERVER: &str = "https://static.rust-lang.org";
 pub(crate) static DEFAULT_DIST_ROOT: &str = "https://static.rust-lang.org/dist";
 
 const TOOLSTATE_MSG: &str =
-    "If you require these components, please install and use the latest successful build version,\n\
+    "If you require the component(s), please install and use the latest successful build version,\n\
```

Comment:
It's a little hard to say without full context, but I think "these components" would read better?

Reply:
@djc Fixed!